Controller/Api/GalleryController.php

```diff
@@ -423,10 +422,19 @@ protected function handleWriteGalleryhasmedia(GalleryInterface $gallery, MediaIn
             $this->galleryManager->save($gallery);
 
             $view = FOSRestView::create($galleryHasMedia);
-            $serializationContext = SerializationContext::create();
-            $serializationContext->setGroups(array('sonata_api_read'));
-            $serializationContext->enableMaxDepthChecks();
-            $view->setSerializationContext($serializationContext);
+
+            // BC for FOSRestBundle < 2.0
+            if (method_exists($view, 'setSerializationContext')) {
```

Comment:
The reverse would be better.

AFAIK, `setSerializationContext` is still here on v2.0, but deprecated.